tests/js/jest.config.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
 	setupFilesAfterEnv: [
 		'<rootDir>/node_modules/@wordpress/jest-preset-default/scripts/setup-test-framework.js',
 		'<rootDir>/tests/js/jest-extensions-setup.js',
+		'expect-puppeteer',
```

Comment:
This line makes `expect( xxx ).toClick( selector );` to work locally on the E2E tests.